tools/seistech_utils/seistech_utils/src/utils.py

```diff
@@ -1,3 +1,5 @@
+import os
+
```

Comment:
You do not need an extra line between `import os` and `from pathlib....` they are all Python built-in stuff. :+1: